### PR TITLE
Add .github/** as a path in CODEOWNERS

### PR DESCRIPTION
**Pull request by chainchad**
Contributors were determined loosely via:

```bash
$ git log --pretty=format:"%h%x09%an%x09%ad%x09%s" --since="2021-02-24" -- .github/ \
  | awk '{print $2}' | sort |  uniq -c | sort -nr
```

### CODE REVIEW

**Review by HenryNguyen5 — COMMENTED** [review]
LGTM. Will let others take a look before merging

**Comment by chainchad** [comment]
Please let me know if I've missed anyone who should be considered a CODEOWNER for .github/** or if anyone wants to be removed.

**Review by jmank88 — APPROVED** [review]

**Comment by chainchad** [comment]
@samsondav if it LGTY, you up for a merge?

**Pull request merged**
